source/_integrations/zha.markdown

```diff
@@ -156,6 +156,39 @@ If you are use ZiGate or Sonoff ZBBridge you have to use some special usb_path c
 - Wifi Zigate : `socket://[IP]:[PORT]` for example `socket://192.168.1.10:9999`
 - Sonoff ZBBridge : `socket://[IP]:[PORT]` for example `socket://192.168.1.11:8888`
 
+### deCONZ Devices
+
+There are two options to connect to deCONZ devices. Those can be plugged into the local host system or a remote
+machine can be used as a serial port forwarder via TCP.
+
+#### Local device
+
+Set the serial device path to the serial port in `/dev/serial/by-id` as described above.
+
+#### Remote device
+
+**Note:** There are no security mechanisms. Anyone with access to your network can take control of your deCONZ device!
+
+Use a device path of the format `tcp://<host>:<port>` to connect to a serial device that is forwarded
+through a remote device. On the remote machine tools like `ser2net` or `socat` can be used to forward
+a serial device to a tcp socket.
+
+Example configuration steps:
+
+- Install `ser2net` from your distributions repository on the remote machine
+- Enable the service to automatically start (usually `sudo systemctl enable ser2net.service`)
+- Add the following single line to `/etc/ser2net.conf`:
+
+```text
+<port>:raw:0:/dev/serial/by-id/<your-serial-device>:38400 NONE 1STOPBIT -XONXOFF NOBREAK
+
+# example:
+23344:raw:0:/dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2457780-if00:38400 NONE 1STOPBIT -XONXOFF NOBREAK
+```
```

Comment:
I honestly, strongly believe, we should discourage the use of this; not advertise it. While possible, it is known not to be the most stable solution.

Reply:
I'm on the fence about this. Even though ConBee sticks are less affected by network connection stability, given the past statistics this has the potential to generate "noise" issue reports, just because people think running this over wifi is a terrific idea.

I'm leaning more toward: let's have it, but not advertise it

Reply:
I am using this setup without problems.
> Tested and working for 2 weeks over a VPN connection. Daily disconnections by the ISP, short-term high latency and other connections issues did not cause any major issues.
Do you mean ser2net is not stable or something else?

Reply:
I would push this to a community guide instead.
Putting it in the official documentation: advertises it.